Init document for open weather connector

ndkhanh-axonivy opened the pull request.

ndkhanh-axonivy commented:
Hi @andreasbalsiger I created this pull request for the documentation step.
Could you help review this README.md file, it describes the purposes and functionality for the OpenWeather connector.
Thanks!

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+# Open Weather Connector
+
+Harness the power of freely available weather data in your applications with the Open Weather Connector! This tool streamlines access to OpenWeatherMap's comprehensive datasets, offering a wealth of valuable information within their free plan:
```
Excellent text, I would slightly change it to: 

Harness the power of freely available weather data in your business processes with OpenWeather! This Axon Ivy connector streamlines access to OpenWeatherMap's comprehensive datasets, offering a wealth of valuable information within their free plan:

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+# Open Weather Connector
```
If I am not wrong, the correct way to write it is: OpenWeather (no space). Can you please make sure to make these changes overall?

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+* **Geocoding:**
+	* **Address to coordinates:** Convert physical addresses to precise latitude and longitude values.
+	* **Coordinates to address:** Obtain the corresponding address for a given latitude and longitude.
+
```
Add:
* Supports you with an easy-to-copy demo implementation to reduce your integration effort.

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+	* **Address to coordinates:** Convert physical addresses to precise latitude and longitude values.
+	* **Coordinates to address:** Obtain the corresponding address for a given latitude and longitude.
+
+<br>
```
I think you can delete this <br> isn't it?

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+
+## Demo
+
+Within the demo, users can access to retrieve comprehensive **5-day weather forecasts** for any global location. Follow these straightforward steps:
```
Change it to:
Within this demo, users can access comprehensive **5-day weather forecasts** for any location worldwide. Follow these simple steps:

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+## Setup
+
+### Application id
+The use of the Open Weather Map API is not free of charge. For development purposes there is a free version with minimal API calls. To use the connector, you must select a suitable API package via the [Open Weather Map API Developer](https://openweathermap.org/api) and generate a **api key**.
```
The OpenWeatherMap weather API is not free to use. However, there is a free version with minimal API calls for development purposes. To use the connector, select a suitable API package via the [OpenWeatherMap API Developer](https://openweathermap.org/api) and generate an **API key**.

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+### Application id
+The use of the Open Weather Map API is not free of charge. For development purposes there is a free version with minimal API calls. To use the connector, you must select a suitable API package via the [Open Weather Map API Developer](https://openweathermap.org/api) and generate a **api key**.
+
+##### How to get api key
```
#### How to get an API key

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+
+## Setup
+
+### Application id
```
### Application ID

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+The use of the Open Weather Map API is not free of charge. For development purposes there is a free version with minimal API calls. To use the connector, you must select a suitable API package via the [Open Weather Map API Developer](https://openweathermap.org/api) and generate a **api key**.
+
+##### How to get api key
+1. Login and go to your [Open Weather Map API keys page](https://home.openweathermap.org/api_keys)
```
1. Login and go to your [OpenWeatherMap API keys page](https://home.openweathermap.org/api_keys)

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+
+##### How to get api key
+1. Login and go to your [Open Weather Map API keys page](https://home.openweathermap.org/api_keys)
+2. Add your api key name and generate it:
```
2. Add your API key name and generate it:

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+1. Login and go to your [Open Weather Map API keys page](https://home.openweathermap.org/api_keys)
+2. Add your api key name and generate it:
+![Register key](images/register-api-key.png)
+3. Api key now is available:
```
3. API key now is available:

andreasbalsiger commented on open-weather-connector-product/README.md:
```diff
@@ -0,0 +1,40 @@
+3. Api key now is available:
+![Register key](images/register-successful.png)
+
+After a **api key** is available, you can store it in your project in the variables.yaml as the variable "openWeatherConnector.appId":
```
After a **API key** is available, you can store it in your project in the variables.yaml as the variable "openWeatherConnector.appId":

ndkhanh-axonivy commented:
Hi @andreasbalsiger , I've just pushed a new commit that addresses the feedback you provided. Could you please review the updated README again?

Thanks for taking a look at the document!

andreasbalsiger reviewed (commented):
Thanks... I can approve the changes.
